src/explorer/webview.ts

```diff
@@ -916,24 +878,26 @@ export class ExplorerWebview {
 
     /**
      * Creates the Tabs Container content, the tabs at the top of the tab container
-     * @param hasCodeFlows Flag to include the CodeFlow tab in the set of tabs
-     * @param hasAttachments Flag to include the Attachments tab in the set of tabs
      */
-    private createTabHeaderContainer(): HTMLDivElement {
+    private createTabHeaderContainer(resultInfo: ResultInfo): HTMLDivElement {
```

Comment:
Simply because during my testing I hit a number of cases where these variables were not properly cleared\set while change the selection. This was causing exceptions and the web-view to not load.

So, it was impeding my testing, so I fixed it :)

---
In reply to: [405131100](https://github.com/microsoft/sarif-vscode-extension/pull/218#discussion_r405131100) [](ancestors = 405131100)